Automattic/simplenote-macos
# Fix: Multibyte Characters going AWOL

### Fix
In this PR we're fixing an issue affecting macOS versions prior to Catalina, in which multibyte characters are disappearing.

Closes #416

### Test: Emojis
1. Log into your account
2. Create a new note
3. Insert 😃
4. Insert ☺️

- [x] Verify both emojis show up side by side!
- [x] Verify the cursor reacts as you would normally expect!

### Test: Sinhala
0. Install the **Sinhala** keyboard on your system
1. Log into your account
2. Create a new note
3. Type (random) characters
4. Wait for a split second until the note synchronizes on the backend
5. Edit and type (random) characters, in between other (random) characters

- [x] Verify every single keystroke is onscreen
- [x] Verify the cursor reacts as you would normally expect!

**If possible** repeat this test in Catalina and (any OS) prior to Catalina. High Sierra / Mojave would be awesome!

### Release
`RELEASE-NOTES.txt` was updated in 8216808 with:

> Fixed an issue affecting multibyte characters
